src/stores/networks.ts

```diff
@@ -0,0 +1,82 @@
+import { RemovableRef, useStorage } from '@vueuse/core';
+import { defineStore } from 'pinia';
+import { createNetwork } from 'src/config/NetworkFactory';
+import { Chain } from 'src/types/Chain';
+export interface NetworksStateInterface {
+    networks: Chain[],
+    currentNetworkName: RemovableRef<string>,
+    preferredNetworkName: RemovableRef<string>,
+}
+const CHAIN_LOCAL_STORAGE = 'selectedChainName';
+const PREFERRED_CHAIN_LOCAL_STORAGE = 'preferredChainName';
+const SUPPORTED_NETWORKS: string[] = process.env.SUPPORTED_NETWORKS.split(' ');
```

Comment:
I would suggest using 'camelCase' in this case not only to differentiate it from the actual environment constant but also because it is already a local variable with a different type. What do you think?

```suggestion
const supportedNetworks: string[] = process.env.SUPPORTED_NETWORKS.split(' ');
```

Reply:
This is also a change request

Reply:
I think it makes sense! I will change that!